Gems/Atom/Utils/Code/Include/Atom/Utils/ImGuiTransientAttachmentProfiler.inl

```diff
@@ -24,23 +24,33 @@ namespace AZ
             {
                 if (ImGui::TreeNodeEx("Memory", ImGuiTreeNodeFlags_DefaultOpen))
                 {
+                    RHI::TransientAttachmentStatistics::MemoryUsage memoryUsage{};
+
+                    for (auto& [deviceIndex, deviceStatistics] : statistics)
+                    {
+                        memoryUsage = deviceStatistics.m_reservedMemory;
```

Comment:
Is this going to be changed to account for all devices?

Reply:
most likely yes